crates/freeze/src/types/conversions.rs

```diff
@@ -99,3 +99,121 @@ impl ToVecHex for OptionVec<RawBytes> {
         }
     }
 }
+
+/// Trait for converting binary data to specific types
+pub trait FromBinaryVec {
+    /// Convert from Vec<Option<RawBytes>> to Self
+    fn from_binary_vec(data: Vec<Option<RawBytes>>) -> Result<Self, CollectError>
+    where
+        Self: Sized;
+}
+
+/// Implementation for Vec<U256>
+impl FromBinaryVec for Vec<U256> {
```

Comment:
remove implementation for `Vec<U256>` and `Vec<I256>`,
add `FromBinary::from_binary(value: RawBytes)` for I256/U256, and add `FromBinaryVec` for `Vec<Option<T>>`